.taskcluster.yml

```
@@ -5,23 +5,112 @@ version: 1
+      then: ${event.pull_request.head.repo.html_url}
+      else: ${event.repository.html_url}
+
+    github_repository_full_name:
+      $if: 'tasks_for == "github-pull-request"'
+      then: ${event.pull_request.base.repo.full_name}
+      else: ${event.repository.full_name}
+
+  in:
+    $let:
+      short_head_branch:
+        $if: 'head_branch[:10] == "refs/tags/"'
+        then: {$eval: 'head_branch[10:]'}
+        else:
+          $if: 'head_branch[:11] == "refs/heads/"'
+          then: {$eval: 'head_branch[11:]'}
+          else: ${head_branch}
+
+      assume_scope_prefix: assume:repo:github.com/${github_repository_full_name}
+    in:
+      $let:
+        default_task_definition:
+          schedulerId: taskcluster-github
+          created: {$fromNow: ''}
+          deadline: {$fromNow: '2 hours'}
+          expires: ${expires_in}
+          provisionerId: aws-provisioner-v1
+          workerType: github-worker
+          retries: 5
+          payload:
+            maxRunTime: 600   # Decision should remain fast enough to schedule a handful of tasks
+            image: mozillamobile/fenix:1.4
+            command:
+              - /bin/bash
+              - --login
+              - -cx
+              - >-
+                git clone ${repository} /build
+                && cd /build
+                && git config advice.detachedHead false
+                && git checkout ${head_branch}
+                && pip install tox
+                && tox -e "$TOX_ENV"
+            env:
+              SKIP_NETWORK_TESTS: "0"
+          metadata:
+            owner: ${user}@users.noreply.github.com
+            source: ${repository}/raw/${head_rev}/.taskcluster.yml
+      in:
+        $flatten:
+          $map:
+            - short_name: 'py27'
+              image_tag: '2.7'
+            - short_name: 'py36'
+              image_tag: '3.6'
+            - short_name: 'py37'
+              image_tag: '3.7'
+          each(python_version):
+            $let:
+              python_task_definition:
+                payload:
+                  image: python:${python_version.image_tag}
+                  env:
+                    TOX_ENV: ${python_version.short_name}
+            in:
+              - $if: 'tasks_for == "github-pull-request" && event["action"] in ["opened", "reopened", "edited", "synchronize"]'
+                then:
+                  $mergeDeep:
+                    - {$eval: 'default_task_definition'}
+                    - {$eval: 'python_task_definition'}
+                    - scopes:
+                        - ${assume_scope_prefix}:pull-request
+                      metadata:
+                        name: 'mozilla-version - ${python_version.short_name} (Pull Request #${event.pull_request.number})'
+                        description: 'Triggered by [#${event.pull_request.number}](${event.pull_request.html_url})'
+              - $if: 'tasks_for == "github-push" && head_branch[:10] != "refs/tags/"'
```

leading index is required iirc
```suggestion
              - $if: 'tasks_for == "github-push" && head_branch[0:10] != "refs/tags/"'
```